### PR TITLE
11765 maps cities

### PR DESCRIPTION
**Pull request by Azaya89**
Added metadata to examples/models/maps_cities.py
- [x] issues: #11765
- [x] tests added / passed

### CODE REVIEW

File: examples/models/maps_cities.py [hunk 1]
```diff
@@ -1,3 +1,12 @@
+''' A plot showing a map of the world, highlighting cities where the population is over 5,000 people, made using the 'GMapPlot' class.
```
Comment (bryevdv) [inline examples/models/maps_cities.py]: update for line length and code-formatting for GMapPlot
```suggestion
''' A plot showing a map of the world, highlighting cities where the population
is over 5,000 people, made using the ``GMapPlot`` class.
```

File: examples/models/maps_cities.py [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+
+.. bokeh-example-metadata::
+    :sampledata: world_cities
+    :apis: bokeh.models.glyphs.Circle, bokeh.models.map_plots.GMapPlot, bokeh.models.tools.PanTool, bokeh.models.tools.WheelZoomTool
```
Comment (bryevdv) [inline examples/models/maps_cities.py]: @Azaya89 at this point we only ever intend to ever show top level `bokeh.models` and avoid any user-facing use of submodules of `bokeh.models`
```suggestion
    :apis: bokeh.models.Circle, bokeh.models.GMapPlot, bokeh.models.PanTool, bokeh.models.WheelZoomTool
```

File: examples/models/maps_cities.py [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+    :refs: :ref:`_ug_styling_plots_axes_labels`, :ref:`_ug_topics_geo`, :ref:`_ug_basic_data`
```
Comment (bryevdv) [inline examples/models/maps_cities.py]: geo ref could be more specific
```suggestion
    :refs: :ref:`_ug_styling_plots_axes_labels`, :ref:`_ug_topics_geo_google_maps`, :ref:`_ug_basic_data`
```

**Review by bryevdv — CHANGES_REQUESTED** [review]
Thanks @Azaya89 a few small comments

File: examples/models/maps_cities.py (hunk 1 above)
Comment (Azaya89) [reply examples/models/maps_cities.py]: Done.

File: examples/models/maps_cities.py (hunk 3 above)
Comment (Azaya89) [reply examples/models/maps_cities.py]: Done

File: examples/models/maps_cities.py (hunk 2 above)
Comment (Azaya89) [reply examples/models/maps_cities.py]: Done

File: examples/models/maps_cities.py [hunk 4]
```diff
@@ -1,3 +1,13 @@
+''' A plot showing a map of the world, highlighting cities where the population
+is over 5,000 people, made using the ``GMapPlot`` class.
+
+.. bokeh-example-metadata::
+    :sampledata: world_cities
+    :apis: bokeh.models.Circle, bokeh.models.GMapPlot, bokeh.models.PanTool, bokeh.models.WheelZoomTool
+    :refs: :ref:`_ug_styling_plots_axes_labels`, :ref:`_ug_topics_geo_google_maps`, :ref:`_ug_basic_data`
```
Comment (bryevdv) [inline examples/models/maps_cities.py]: sorry for confusion: Sphinx uses the leading `_` when defining the anchors, but it is not present when using the ref (i.e. here)
```suggestion
    :refs: :ref:`ug_styling_plots_axes_labels`, :ref:`ug_topics_geo_google_maps`, :ref:`ug_basic_data`
```

Comment (Azaya89) [reply examples/models/maps_cities.py]: Done.

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @Azaya89 !

**Pull request merged**